.travis.yml

```diff
@@ -1,9 +1,33 @@
 language: python
 python:
-  - "2.6"
```

Comment:
This matches setup.py as discussed https://github.com/ArduPilot/pymavlink/pull/10#issuecomment-251911149